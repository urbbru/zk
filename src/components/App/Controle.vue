<template>
    <h2 class="mt-5">Controleren</h2>
    <div class="form-group">
        <h3>Gekozen pakket</h3>
    </div>
    <div class="form-group">
        <h3>Totaalpremie</h3>
    </div>
    <div class="form-group">
        <h3>Adres en contactgegevens</h3>
    </div>
</template>
