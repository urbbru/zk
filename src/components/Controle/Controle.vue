<template>
    <h2 class="mt-5">Controleren</h2>
    <div class="form-group">
        <h3>Gekozen pakket</h3>
        <h4>
            {{ aanmeldenFormStore.gegevens.naam }} ({{
                aanmeldenFormStore.gegevens.geboortedatum
            }})
        </h4>
        <p>{{ aanmeldenFormStore.verzekering.basisVerzekering }}</p>
        <p>{{ aanmeldenFormStore.verzekering.eigenRisico }}</p>
        <p>{{ aanmeldenFormStore.verzekering.aanvullendeVerzekering }}</p>
    </div>
    <div class="form-group">
        <h3>Totaalpremie</h3>
    </div>
    <div class="form-group">
        <h3>Adres en contactgegevens</h3>
    </div>
    <button class="btn" @click="goToVerzekering">Terug naar gegevens</button>
</template>

<script>
import { useAanmeldenFormStore } from '../../stores/aanmeldenForm';

export default {
    setup() {
        const aanmeldenFormStore = useAanmeldenFormStore();

        function goToVerzekering() {
            aanmeldenFormStore.goToPreviousStep();
        }

        return {
            aanmeldenFormStore,
            goToVerzekering
        };
    }
};
</script>
