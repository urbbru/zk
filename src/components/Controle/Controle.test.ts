import { it, describe, expect } from 'vitest';
import { shallowMount } from '@vue/test-utils';
import Controle from './Controle.vue';

describe('Controle.vue', () => {
    it('Should render correctly', () => {
        const wrapper = shallowMount(Controle);

        expect(wrapper.element).toMatchSnapshot();
    });
});
