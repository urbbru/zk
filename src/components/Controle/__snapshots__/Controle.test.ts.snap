// Vitest Snapshot v1

exports[`Controle.vue > Should render correctly 1`] = `
<div
  data-v-app=""
>
  
  <h2
    class="mt-5"
  >
    Controleren
  </h2>
  <div
    class="form-group"
  >
    <h3>
      Gekozen pakket
    </h3>
  </div>
  <div
    class="form-group"
  >
    <h3>
      Totaalpremie
    </h3>
  </div>
  <div
    class="form-group"
  >
    <h3>
      Adres en contactgegevens
    </h3>
  </div>
  
</div>
`;
